litestar/openapi/spec/schema.py

```diff
@@ -641,6 +647,14 @@ class Schema(BaseSchemaObject):
     def __hash__(self) -> int:
         return _recursive_hash(self)
 
+    @classmethod
+    def field_aliases(cls) -> dict[str, str]:
```

Comment:
Since this is kind of in a hot path, I think it would make sense to cache it

Reply:
Addressed via `cls._field_aliases` in newer revisions.

Reply:
Modified accordingly.